index.html

```diff
@@ -58,121 +63,140 @@ <h3>Let's connect</h3>
+            <ul class="work-categories margin-top-12 padding-0">
+              <li><span>html</span></li>
+              <li><span>css</span></li>
+              <li><span>javascript</span></li>
             </ul>
+            <button type="button" class="button button-enabled">See Project</button>
           </div>
-          <p class="work-description text-color-primary margin-top-20">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <ul class="work-categories margin-top-12 padding-0">
-            <li><span>html</span></li>
-            <li><span>css</span></li>
-            <li><span>javascript</span></li>
-          </ul>
-          <button type="button" class="button button-enabled">See Project</button>
         </div>
       </div>
     </section>
 
-    <section class="about-myself padding-top-114">
+    <section id="about" class="about-myself padding-top-114">
       <div class="flex about-wrap">
-        <h2 class="section-title">About myself</h2>
-        <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <a href="#" class="headline-cta margin-top-12">
-          <h3>Let's connect</h3>
-        </a>
-        <ul class="social-icons margin-top-12">
-          <li>
-            <a href="#">
-              <img class="social twitter" src="images/icons/twitter-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social linkedin" src="images/icons/linkedin-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social m" src="images/icons/m-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social github" src="images/icons/github-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img class="social f" src="images/icons/f-icon.png" alt="Twitter icon">
-            </a>
-          </li>
-        </ul>
-        <button type="button" class="button button-enabled">See my resume</button>
+        <div class="about-details-wrap">
+          <h2 class="section-title">About <br> myself</h2>
+          <p class="headline-profile text-color-primary margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <a href="#" class="headline-cta margin-top-12">
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) design as much as possible. Please make sure that the skills section contains approx 60% of the total width and the left section contains the remaining and it should match with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165695894-84f92325-bb19-4a2a-8c4a-772aed4e1a80.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165695989-dfe79227-4790-4bba-ab02-d33f5f96cfe6.png)